### DISCUSSION TITLE
YMConnect for MotoSimEG-VRC or equivalent SDK?

### DISCUSSION THREAD

**Discussion by GithubCHSs** [Yaskawa-Global/YMConnect#13] [Q&A] [answered]
Hello everyone

Is there any possibility to read/write IO from virtual controller in MotoSimEG-VRC?

It would be great if there was any tool (SDK) to write your own APP for this kind of communication. This would make it possible to cooperate with third-party programs and allow users to check their rob applications.

**Comment by ted-miller** [comment, marked as answer]
My apologies for missing this question until now.

Today, there are no tools to programmatically interact with MotoSim. This topic is being worked on.
